Conference-Android/app/src/main/java/com/sagetech/conference_android/app/ui/activities/NewEventDetailActivity.java

```diff
@@ -25,6 +28,20 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            // Respond to the action bar's Up/Home button
+            case android.R.id.home:
+                Intent intent = NavUtils.getParentActivityIntent(this);
```

Comment:
this probably should change back if it is relating to what I spoke of earlier in the AndroidManifest.xml